website/content/en/docs/contribution-guidelines/developer-guide.md

```diff
@@ -113,6 +113,36 @@ COPY watches.yaml ${HOME}/watches.yaml
 COPY helm-charts/ ${HOME}/helm-charts/
 ```  
 
+## How the operator-sdk images are built?
+
+In the Deploy stage configured in the [.travis.yml ][travis], the images are built and pushed to the `quay.io/operator-framework` repository. This stage is just executed against the master branch when a Pull Request is merged. 
+
+See that the images are built with different architectures from master as it is also done in the release process. See in the release Tab that each version has different architectures as well for each supported operator type such as Helm and Ansible for example. 
+
+In the release process, the script [.release.sh][release-sh] is executed and it will call the [makefile][makefile] target [make release](https://github.com/operator-framework/operator-sdk/blob/master/Makefile#L113) which will call the same targets which are used by the Travis in the master CI job to build these images. To know more about the release process, see the [doc][release-doc].
+
+## How to test the built of operator-sdk images? 
+
+If you are looking for to contribute to it, then you will need to ensure that the changes made in your Pull Request will also be executed successfully in the `Deploy` stage which, as described above, will just be trigged from the master branch.
+
+So, you will need to enable the Travis in your fork repository and perform a few configurations to allow it to be executed and check it. Following the steps. 
+
+- Create image repos in quay (or another registry that supports multi-arch images) for ansible, helm, and scorecard proxy. For each image type, you need repos for the manifest list and one for each architecture (e.g. `ansible-operator`, `ansible-operator-amd64`, `ansible-operator-s390x`, etc.)
+- Set the following environment variables in the Travis settings for your fork:
+
+    - `ANSIBLE_IMAGE` docker image name (e.g. `quay.io/joelanford/ansible-operator`)
+    - `HELM_IMAGE` same as above, but for helm
+    - `SCORECARD_PROXY_IMAGE` same as above, but for scorecard proxy
+    - `DOCKER_USERNAME` credentials for your repo
+    - `DOCKER_PASSWORD` credentials for your repo
+    - `DOCKER_CLI_EXPERIMENTAL`  set to `enabled` 
+    - `COVERALLS_TOKEN`  token to integrate the project with `https://coveralls.io/`. So, enable your fork in and generate a token to allow it. 
```

Comment:
enable your fork in Travis? Or coveralls?

Reply:
Since this is a list of steps, move "enable your fork in coveralls" to its own step